extensions/amp-lightbox/0.1/amp-lightbox.js

```diff
@@ -152,6 +160,8 @@ class AmpLightbox extends AMP.BaseElement {
       }
       // TODO: instead of laying out children all at once, layout children based
       // on visibility.
+      this.element.addEventListener('transitionend', this.boundReschedule_);
+      this.element.addEventListener('animationend', this.boundReschedule_);
       this.scheduleLayout(container);
```

Comment:
Yes. would still be useful for elements with `relayoutNeeded = true`. amp-img for example.